BUILDING.md

```diff
@@ -19,7 +19,7 @@ The following tools are required to build RPCS3 on Windows 10 or later:
     with standalone **CMake** tool.
 
 - [Python 3.6+](https://www.python.org/downloads/) (add to PATH)
-- [Qt 6.7.3](https://www.qt.io/download-qt-installer)
+- [Qt 6.7.3](https://www.qt.io/download-qt-installer) with the "qtmultimedia" module (In case you can't download from the official site, you can use [Another Qt installer](https://github.com/miurahr/aqtinstall))
```

Comment:
Its in the components of the installer

Reply:
> Its in the components of the installer

I guess i should specify that it's for those, who don't use the official installer - when i installed qt through aqt, i had to manually add the qtmultimeida module.

Reply:
It is in the official I am pretty sure, there is a dropdown options for components types, I installed 6.7.3 today

Reply:
There is no harm in linking an alternative installer I guess